src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -554,8 +554,9 @@ define(function (require, exports, module) {
          * @private
          * @type {Droppable~onDragTargetMove}
          */
-        _handleDragTargetMove: function (draggedLayers, dragPosition) {
-            var isDropTarget = !draggedLayers.includes(this.props.layer),
+        _handleDragTargetMove: function (draggedLayerIds, dragPosition) {
+            var draggedLayers = this.props.document.layers.byIDs(draggedLayerIds),
```

Comment:
You can also filter afterwards to make sure layer model exists, but I'm not a fan of adding yet another method to layerstructure that can be done with a map and a filter, and it will keep things Immutable.